modules/backend/layouts/auth.htm

```diff
@@ -11,6 +11,7 @@
         <link href="<?= Url::asset('modules/system/assets/ui/storm.css') ?>" rel="stylesheet">
         <link href="<?= Url::to('modules/backend/assets/css/october.css') ?>" rel="stylesheet">
         <script src="<?= Url::to('modules/backend/assets/js/vendor/jquery.min.js') ?>"></script>
+        <script src="<?= Url::to('modules/backend/assets/js/vendor/jquery-migrate.min.js') ?>"></script>
```

Comment:
Doesn't this load it twice when loading it here and in october.js (compiled in october.min.js)?

Reply:
I wasn't sure if I had to add it to both? Probably best to not add it to october.js

- Just add it to only _head.htm and auth.htm

Reply:
@LukeTowers 

Having **jQuery Migrate** load in files: `_head.htm` and `auth.htm` doesn't seem to load the file twice. See my screenshot above showing the file only being loaded once. Also the original files loaded **jQuery.min.js** in both files and they are not loaded twice. 

I have removed the **jQuery Migrate** being loaded in the `october.js` / `october.min.js` file(s) to avoid double loading see commit: https://github.com/octobercms/october/pull/3238/commits/decef37fd3863af861c421b1fd790dce8d31b8f1

I can't see any problems with this code now? Been running it in a test environment for several days with many plugins from lots of users and have not come across any issues so far.

I wait for your review and comments. Let me know if you need me to do anything else? Thanks.